src/server/node.rs

```diff
@@ -136,7 +136,7 @@ where
             meta.store_id = Some(store_id);
         }
         if let Some(first_region) = self.check_or_prepare_bootstrap_cluster(&engines, store_id)? {
-            info!("try bootstrap cluster"; "store_id" => store_id, "region" => ?first_region);
+            debug!("try bootstrap cluster"; "store_id" => store_id, "region" => ?first_region);
```

Comment:
Hmm, I think it's reasonable to log the bootstrap procedure. It's a quite important event, should not happen silently.

Reply:
Ok, PTAL at #5172 !